package.json

```diff
@@ -230,6 +231,21 @@
         "title": "Create Module",
```

Comment:
Please move Create module context menu to module label

Reply:
fixed